src/services/nexboard/utils/Nexboard.js

```diff
@@ -0,0 +1,139 @@
+const rp = require('request-promise-native');
+
+const DEFAULT_BASE_URL = 'https://nexboard.nexenio.com/portal/api/v1/public/';
+
+class Nexboard {
+	constructor(apiKey, userID, url) {
+		this.apiKey = apiKey;
+		this.user = userID;
+		this.url = url || DEFAULT_BASE_URL;
+	}
+
+	getProject(projectId) {
+		const settings = {
+			method: 'GET',
+			uri: `${this.url}projects/${projectId}`,
+			qs: {
+				token: this.apiKey,
+				userId: this.user,
+			},
+			json: true,
+		};
+
+		return rp(settings);
+	}
+
+	getProjectsIds() {
+		const settings = {
+			method: 'GET',
+			uri: `${this.url}projects`,
+			qs: {
+				userId: this.user,
+				token: this.apiKey,
+			},
+			json: true,
+		};
+
+		return new Promise((resolve, reject) => {
+			rp(settings)
+				.then(res => resolve(res.map(e => e.id)))
+				.catch(err => reject(new Error(
```

Comment:
Please use feather errors package to create errors, like throw new BadRequest('message', err); And if message go out "hide" err for extern users and pass err with winston logger to intern logs.